packages/jsonrpc_client/pubspec.yaml

```diff
@@ -9,12 +9,12 @@ environment:
 dependencies:
   analyzer: ">=4.4.0 <6.0.0"
   build: ^2.1.1
-  http: ^0.13.4
+  http: ^1.1.0
   solana:
     path: ../solana
   source_gen: ^1.1.1
 
 dev_dependencies:
-  dart_code_metrics: ^5.7.3
+  # dart_code_metrics: ^5.7.3
```

Comment:
It shouldn't be commented.

Reply:
Does dart_code_metrics support http 1.1.0 ?

Reply:
I was hoping if I could point to a version of dart_code_metrics which supports http 1.1.0, if it doesnt then we will need to upgrade its dependency as well but I cannot find the repo for dart_code_metrics

Reply:
Awesome. I have synced the latest code.